### PR TITLE
Fix Percona XtraDB Cluster (mysql) annotations

### PR DESCRIPTION
**Pull request by pmacik**
Signed-off-by: Pavel Macík <pavel.macik@gmail.com>

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
  included if any changes are user facing
- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
- [ ] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)

### CODE REVIEW

File: pkg/binding/registry/registry.go [hunk 1]
```diff
@@ -31,15 +31,19 @@ func New() Registry {
 			},
 			schema.GroupVersionKind{Group: "pxc.percona.com", Version: "v1-8-0", Kind: "PerconaXtraDBCluster"}: {
 				"service.binding/type":     "mysql",
+				"service.binding/database": "mysql",
```
Comment (baijum) [inline pkg/binding/registry/registry.go:34]: This is the name of the database, right?
Is that going to be always `mysql`?

File: test/acceptance/features/supportExistingOperatorBackedServices.feature [hunk 2]
```diff
@@ -169,18 +169,17 @@ Feature: Support a number of existing operator-backed services out of the box
...
 
 
-  @crdv1beta1
   Scenario: Bind test application to Mysql provisioned by Percona Mysql operator
     Given Percona Mysql operator is running
     * Generic test application is running
     * The Custom Resource is present
           """
-          apiVersion: pxc.percona.com/v1-8-0
+          apiVersion: pxc.percona.com/v1-10-0
```
Comment (sadlerap) [inline test/acceptance/features/supportExistingOperatorBackedServices.feature:177]: Will this work?  We don't have `v1-10-0` as a recognized GroupVersionKind in our registry, so we're not using any the new behavior listed there.

Comment (baijum) [inline test/acceptance/features/supportExistingOperatorBackedServices.feature:177]: I don't see annotation for this API version in the registry.

**Review by sadlerap — CHANGES_REQUESTED** [review]
If we want to support `v1-10-0` resources, I think we need to add it as a recognized version in our registry.

File: test/acceptance/features/supportExistingOperatorBackedServices.feature (hunk 2 above)
Comment (pmacik) [reply test/acceptance/features/supportExistingOperatorBackedServices.feature:177]: good point - fixed

Comment (pmacik) [reply test/acceptance/features/supportExistingOperatorBackedServices.feature:177]: it is now... I added it

**Comment by baijum** [comment]
<rant>
API version tied to the software version looks weird!
</rant>

File: pkg/binding/registry/registry.go (hunk 1 above)
Comment (sadlerap) [reply pkg/binding/registry/registry.go:34]: For this particular `GroupVersionKind`, I think it's always going to be a mysql db that gets created.  I see no reason for Percona to retroactively change this to be something else.

**Review by sadlerap — APPROVED** [review]

**Pull request merged**